qiskit_sphinx_theme/furo/base/static/styles/qiskit_changes.css

```diff
@@ -1,26 +1,54 @@
 /* Define some Furo variables. See
  *  https://github.com/pradyunsg/furo/tree/main/src/furo/assets/styles/variables
- *  for all options. */
+ *  for all options.
+ *
+ * Variables that start with `--qiskit-` are new values we use in this file; they are not overrides.
+ */
 body {
     --color-brand-primary: #8A3FFC;
     --color-brand-content: #8A3FFC;
     --font-stack: 'IBM Plex Sans', 'Roboto', 'Helvetica Neue', 'Arial', sans-serif;
     --font-stack--monospace: 'IBM Plex Mono', 'Consolas', 'Courier New', monospace;
+    /* This value is duplicated from `top-nav-bar.js`. Its definition of the variable is not
+     * exposed globally. Keep in sync. */
+    --qiskit-top-nav-bar-height: 3.5rem;
```

Comment:
See https://github.com/Qiskit/qiskit_sphinx_theme/pull/295 for an explanation of why we take this approach.

Unlike https://github.com/Qiskit/qiskit_sphinx_theme/pull/295, this uses a more unique variable name so that we don't override Furo's `--header-height`.